chapter_convolutional-neural-networks/channels_vn.md

```diff
@@ -127,7 +127,16 @@ Reality is a bit more complicated than the most naive interpretations of this in
 So it may not be that a single channel learns an edge detector but rather that some direction in channel space corresponds to detecting edges.
 -->
 
-*dịch đoạn phía trên*
+Cho đến nay, bất kể số lượng kênh đầu vào là bao nhiêu thì ta vẫn luôn kết thúc với chỉ một kênh đầu ra.
+Tuy nhiên, như đã thảo luận trước đây, hóa ra việc có nhiều kênh ở mỗi lớp là rất cần thiết.
+Trong các kiến trúc mạng nơ-ron phổ biến nhất, ta thường tăng kích thước chiều kênh khi tiến sâu hơn trong mạng, đồng thời lấy mẫu xuống để bù lại độ phân giải không gian cho *chiều kênh* sâu hơn này.
+Một cách trực quan, ta có thể xem mỗi kênh tương ứng với một vài đặc trưng khác nhau.
+Nhưng thực tế thì lại phức tạp hơn một chút so với cách diễn giải ngây thơ này vì các biểu diễn không được học độc lập mà được tối ưu hóa để cùng có ích.
```

Comment:
```suggestion
Theo trực giác, ta có thể xem mỗi kênh tương ứng với một vài đặc trưng khác nhau.
Nhưng thực tế phức tạp hơn một chút so với cách diễn giải theo trực giác này vì các biểu diễn không được học độc lập mà được tối ưu hóa để có ích khi kết hợp với nhau.
```

Reply:
Mình sửa suggestion của @cuongvng để thay chữ `some = một vài -> nào đó` nhé.

Reply:
@lkhphuc lk em thấy bản gốc là `some different set of features`, nên 1 kênh không chỉ ứng với 1 đặc trưng.